canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -251,13 +257,20 @@ $toolbar-divider-width: 1px;
 	}
 
 	&.is-in-menu {
-		justify-content: left;
+		justify-content: space-between;
 
 		& .content-main {
 			padding: 8px 0 6px;
 		}
 	}
 
+	& .checkmark {
+		pointer-events: none;
+		padding-top: 5px;
```

Comment:
I dont see a difference setting these top and left padding
<img width="1451" alt="image" src="https://github.com/user-attachments/assets/40dfdddc-dfc5-45f8-9942-b6bb9ce8b457">